### PR TITLE
sys/shell: make dev parameter optional for fibroute

### PR DESCRIPTION
**Pull request by cgundogan**
1. The first commit refactors the code to add an entry into a separate function
2. The second commit adds the option to add entries without specifying a device, **IF:**
  * `MODULE_NG_NETIF` exists
  * only one interface is configured

With this PR it is possible to shorten
`fibroute add 2001:db8::1 via fe80::1 dev 7`
`fibroute add 2001:db8::2 via fe80::2 dev 7 lifetime 6000`
to
`fibroute add 2001:db8::1 via fe80::1`
`fibroute add 2001:db8::2 via fe80::2 lifetime 6000`

if only one interface is configured (output of `ifconfig`) and if the `ng_netif` module is used

### CODE REVIEW

**Comment by cgundogan** [comment]
changed scope of commits from `ng_fib` to `sys/shell`.

**Comment by BytesGalore** [comment]
nice one, will have a closer look this afternoon.

File: sys/shell/commands/sc_fib.c [hunk 1]
```diff
@@ -25,12 +25,15 @@
...
+#include "ng_netif.h"
+#endif
 
-#define INFO1_TXT "fibroute add <destination> via <next hop> dev <device>"
+#define INFO1_TXT "fibroute add <destination> via <next hop> [dev <device>]"
 #define INFO2_TXT " [lifetime <lifetime>]"
 #define INFO3_TXT "       <destination> - the destination address\n" \
                   "       <next hop>    - the address of the next-hop towards the <destination>\n" \
-                  "       <device>      - the device id of the Interface to use\n"
+                  "       <device>      - the device id of the Interface to use. Optional if only one interface is available.\n"
```
Comment (OlegHahm) [inline sys/shell/commands/sc_fib.c]: line too long

**Comment by cgundogan** [comment]
addressed @OlegHahm's comment. squashed immediately.

**Comment by cgundogan** [comment]
travis approves. @BytesGalore what's your take on it?

**Comment by BytesGalore** [comment]
Oh, totally forgot to ACK, sry.
I would say please squash and merge at will.

**Comment by cgundogan** [comment]
squashed and travis approves: GO

**Pull request merged**